simplyblock_core/models/nvme_device.py

```diff
@@ -46,19 +46,12 @@ class NVMeDevice(BaseModel):
     nvmf_ip: str = ""
     nvmf_nqn: str = ""
     nvmf_port: int = 0
-    overload_percentage: int = 0
-    partition_jm_bdev: str = ""
-    partition_jm_size: int = 0
-    partition_main_bdev: str = ""
-    partition_main_size: int = 0
-    partitions_count: int = 0
```

Comment:
removing this will also have to needs updating `simplyblock_web/static/swagger.yaml` 


Also these variables are properties within the FoundationDB. So I am not 100% confidant if removing this will preserve backward compatibility. 

So I leave out this variables and handle only the dead functions

Reply:
I reintroduced the variables but marked them as unused.